.golangci.yml

```diff
@@ -0,0 +1,31 @@
+linters:
+  auto-fix: false
+  enable:
+    - nakedret 
+    - interfacer
+    - varcheck
+    - deadcode 
+    - structcheck 
+    - misspell
+    - maligned 
+    - ineffassign
+    - goconst
+    - goimports
+    - errcheck 
+    - dupl 
+    - unparam 
+    - golint 
+    - staticcheck
+    - unused
+    - gosimple
+    # missing to be added
+    # - lll (250), errcheck, gocyclo, gosec, scopelint, unconvert, stylecheck
+run:
+linters-settings:
+  errcheck:
+    check-type-assertions: true
+  lll:
+    line-length: 250
```

Comment:
```suggestion
```

The goal of have `lll` is not allow the line to be bigger than the page (default value) which is bad since requires to use the scroll to check the code. However, was decided that it would be removed. I do not see the point to add it with 250 characters at all in this way. So, IMO we should not add the lll(250).

Reply:
We intentionally removed `lll`: #3362

Reply:
IMO it should be 95 (80 really)

Reply:
while `lll` was left in the config, it wasn't enabled... I will remove from config.  I didn't realize longer lengths were controversial...   I'll leave this article regarding [Torvalds 2020 view of 80 char limits.](https://www.theregister.com/2020/06/01/linux_5_7/#:~:text=Linux%20kernel%20overlord%20Linus%20Torvalds,a%20thing%20of%20the%20past.)